draft-js-mention-plugin/src/utils/getWordAt.js

```diff
@@ -5,7 +5,8 @@ const getWordAt = (string, position) => {
   const pos = Number(position) >>> 0;
 
   // Search for the word's beginning and end.
-  const left = str.slice(0, pos + 1).search(/\S+$/);
+  // Do NOT adjust pos if it already at the end of word.(Happens when mentiontrigger is empty(''))
+  const left = str.slice(0, str.charAt(pos) === ' ' ? pos : pos + 1).search(/\S+$/);
```

Comment:
hi @danyu thx for this pr! Could you make this an if statement rather a ternary expression, this is more compact but I think it should be a little easier to read (for the benefit of us all ❤️ )

Reply:
Sure, does following look better: :sparkles:

```javascript
  // Pos goes to end of word already when mentiontrigger is empty(''),
  // so make sure adjusting pos ONLY when cursor pos is NOT at the end of word.
  if (str.charAt(pos) !== ' ') {
    pos += 1;
  }

  // Search for the word's beginning and end.
  const left = str.slice(0, pos).search(/\S+$/);
```